misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJobQueue.kt

```diff
@@ -141,10 +141,7 @@ internal class SqsJobQueue @Inject internal constructor(
     )
 
     // Preserve original trace id, if available.
-    (span as? DDSpan)?.let {
-      val traceId = it.context().traceId.toString()
-      metadata[SqsJob.JOBQUEUE_METADATA_ORIGINAL_TRACE_ID] = traceId
-    }
+    metadata[SqsJob.JOBQUEUE_METADATA_ORIGINAL_TRACE_ID] = span.context().toTraceId()
```

Comment:
We _may_ want some safety around an empty/null traceID as I think if there isn't an active tracer, this will get a NoopSpan (or something like that) and could publish an empty value.

Something like:

```suggestion
    span.context().toTraceId()?.takeIf { it.isNotBlank() }?.let {
        metadata[SqsJob.JOBQUEUE_METADATA_ORIGINAL_TRACE_ID] = it
    }
```